README.md

```diff
@@ -2,72 +2,128 @@
+sudo apt-get update
+sudo apt-get install nodejs
+sudo apt-get install npm
+```
+
+Now, install git:
 ```bash
 sudo apt-get install git
 ```
 
-## Cloning the repository
+### For Mac Users
+
+Note: Mac usage is for testing purposes only and some perifferals might not work properly.
+
+Check your local node installation by opening a Terminal and writting the following command.
+```bash
+node -v
+```
+
+If your node version is different than this one we suggest using nvm to go back and forth on different node versions. For more information, Visit the [documentation](https://github.com/nvm-sh/nvm#installation-and-update).
+
+Make sure you have git installed:
+```bash
+git --version
+```
+In case you get an error, you need to install git in your computer. We recomend using [homebrew](https://brew.sh/
+) for that. Once have homebrew installed, you can install git as follows:
+```bash
+brew install git
+```
+
+Install command line tools
+```bash
+xcode-select --install
+```
+
+Now your device is set and ready to install the SDK.
+
+---
+
+## Installing the SDK
+
+### Clone the repository
 Clone the repository on your device, for example with ssh, and enter the folder:
 ```bash
-git clone https://github.com/BankingofThings/BoT-NodeJS-SDK.git
-cd BoT-NodeJS-SDK
+git clone https://github.com/BankingofThings/BoT-NodeJS-SDK.git && cd BoT-NodeJS-SDK
 ```
 
-## Creating your environment
+### Creating your environment
 As for now, this SDK supports debian based environments. To setup your environment, run:
 ```bash
 make environment-debian
 ```
-For development you can use macOS with Xcode installed.
+For development wit a Mac, you can skip this step. Make sure you use a macOS with Xcode installed.
 
-## Installing dependencies
+### Installing dependencies
 To install dependencies run:
 ```bash
 make install
 ```
-## Configuration
-Before your first run, start the configuration tool.
+---
+
+### Configuration
+Before your first run, start the configuration tool. It will ask for a few things to configure:
 ```bash
-cd bin
-./config.sh
+sh bin/config.sh
 ```
-### Enter your MakerID
-Copy paste your unique MakerID from the FINN portal. You can find it under "Account"
 
-### Multi-pairable device (yes/no)
+#### Enter your MakerID
+Copy paste your unique MakerID from the account page in the FINN [Maker Portal](https://maker.bankingofthings.io/account).
+
+#### Multi-pairable device (yes/no)
 When your IoT device is meant to be used by more than one user say "yes" to this option.
 This will result in an IoT device that is usable for all FINN app users.
 If your IoT device meant to have one owner then say "no" to this option
 
-### Enter your alternativeID
+#### Enter your alternativeID
 When configuring a Multi-pairable device we have to set an alternativeID.
 This alternativeID needs to be filled-in during the pairing process in the FINN Companion app.
 Examples can be a "loyalty card number" or the "license plate" of your car.
 
+---
```

Comment:
?

Reply:
This is just a separator. I thought it would be nice to mark where the previous section ends and the new one begins